File: NetBeansProjects/WebApplication1/src/java/com/epic/canvascontrollibrary/CCLWindow.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
package com.epic.canvascontrollibrary;

import java.util.ArrayList;
import java.util.List;

/**
 *
 * @author Gilgamesh
 */
    public class CCLWindow
    {
        public String WindowCount;
        public String X;
        public String Y;
        public String Width;
        public String Height;
        public String Depth;
        public String CanvasID;
        public String ParentWindowID;
        public List<Object> ChildWindowIDs;
        public String ControlType;
        public String ControlNameID;
        public String MultiWindowControlsMainWindowId;
        public String TabStopIndex;


        CCLWindow()
        {
            ChildWindowIDs = new ArrayList<Object>();
        }
    }

File: ASP.NET MVC Web Api New Solution/CanvasControlLibrary/CanvasControlLibrary/Models/CCLWindow.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.ComponentModel.DataAnnotations.Schema;

namespace CanvasControlLibrary.Models
{
    public class CCLWindow
    {
        public int WindowCount { get; set; }
        public int X { get; set; }
        public int Y { get; set; }
        public int Width { get; set; }
        public int Height { get; set; }
        public int Depth { get; set; }
        public string CanvasID { get; set; }
        public int ParentWindowID { get; set; }
        public List<int> ChildWindowIDs { get; set; }
        public string ControlType { get; set; }
        public string ControlNameID { get; set; }
        public int MultiWindowControlsMainWindowId { get; set; }
        public object Tag { get; set; }
        public int TabStopIndex { get; set; }

        public CCLWindow()
        {
            ChildWindowIDs = new List<int>();
        }
    }
}